tasks/IsTheRefrigeratorRunning.tex

```diff
@@ -0,0 +1,37 @@
+\section{Is the Refrigerator Running [Housekeeper]}
+The people who live in the arena are so sloppy! The robot finds minor faults in the state of the arena and rectifies them. This is an advanced version of Clean Up for Stage II.
+
+% \subsection{Focus}
+% This test focuses on object perception, manipulation, and planning.
+
+\subsection{Main Goal}
+The robot wanders around the apartment looking for things that are wrong (we will provide a list of potentially-wrong things), and then fixes them. Examples: The refrigerator door is open, the pillows from the bed are on the floor, the television is on with nobody watching.
```

Comment:
As is I don't see this as much different than the `Clean Up` task from Stage I.

The list of potentially-wrong things would need to be provided in this PR for it to be merge-able.

Reply:
This was way more exciting when we discussed it as a group. We sort of had a brainstorming session to get people pumped about 2019, and the tasks that we submitted were a result of that. I'll ping my team to ask for some suggestions here.

Reply:
Cleanup is about identifying objects in a room and classifying them as not-trash vs trash. After detection, the action to be carried out is - if its trash, throw them in the dustbin, else place them where they belong by navigating to the room and place it there. whereas, in this particular task, we propose a different action category, in which the robot interacts with the object - for example, the robot detects the fridge door is open, interacts with the door to close it, detects the television is running, but there's no human in the room to watch/hear the tv, looks for the remote, and switches off the television, sees that the fan/ lights are on but there's nobody in the room so it looks for the switch and turns it off , detects the electric stove is on as identified by the red coil color and theres no vessel on top of it - so it looks for the knob and turns it off. There can be several such examples where the robot needs to interact with the environment to fix it, rather than pick and place the object.

Reply:
I agree with @moriarty in that the setup is similar to `Clean up`, in contrast to `Dishwasher`, `Help me carry`, etc.

On your suggestions:
1. OK
2. Hard, almost impossible in the Arena. I'm not sure we would be able to use the TV there
3. Fan is hard but might be okay, but lights is almost impossible in the arena
4. Don't even think about this
5. ...
    - Main door: doable
    - Window: impossible
    - Cupboard: doable
6. If my robot dares to do that, I dismantle it in the most painful way.